docs/pages/textarea.js

```diff
@@ -53,6 +53,265 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-do-content-length"
+        label="Board description"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        placeholder="What's your board about?"
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use TextArea when the text input is a single, non-sentence response — even in cases with long content. Use [TextField](https://gestalt.pinterest.systems/textfield) instead."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('https://www.pinterest.com/pin/768145280205600341/');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-dont-single-line"
+        label="Destination URL"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use `label` to clearly denote what information the user should input. Use `placeholder` sparingly as [they can erode usability of form fields](https://www.nngroup.com/articles/form-design-placeholders/)."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-do-label"
+        label="Tell everyone what this Pin is about"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Remove the label, as this creates accessibility and usability issues."
+            defaultCode={`
+function Example(props) {
+  const [value, setValue] = React.useState('');
+
+  return (
+    <Box width="100%">
+      <TextArea
+        id="best-practices-dont-placeholder"
+        onChange={({ value }) => {
+          setValue(value);
+        }}
+        placeholder="Tell us your story"
+        value={value}
+      />
+    </Box>
+  );
+}
+            `}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use helperText to provide additional context that will aid the user in most effectively inputing information."
```

Comment:
nit: `helperText` should use backticks, since it's referring to a specific prop

Reply:
Done